src/app/beer_garden/monitor.py

```diff
@@ -3,14 +3,17 @@
 
 These are abstract classes generated to be utilizes for functions based off OS file events
 """
+import logging
 from pathlib import Path
 
-from watchdog.events import PatternMatchingEventHandler
+from brewtils.models import Event, Events, Job
+from watchdog.events import PatternMatchingEventHandler, RegexMatchingEventHandler
 from watchdog.observers.polling import PollingObserver
 
-from beer_garden.db.mongo.models import Event
 from beer_garden.events import publish
 
+logger = logging.getLogger()
```

Comment:
`logger = logging.getLogger(__name__)`

Else it appears in the logs as:
`2024-08-06 08:19:26,211 - root - INFO - Stop dir monitor on`

Instead of:
`2024-08-06 08:19:26,211 - beer_garden.monitor - INFO - Stop dir monitor on`

Reply:
Updated logging in monitor and scheduler.